src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -31,41 +32,74 @@ const MainNavDroplist = Droplist.extend({
         <img class="app-nav__header-logo" src="/rwell-logo.svg" />
       {{else}}
         <div class="u-text--overflow">
-          <h2 class="app-nav__header-title u-text--overflow">{{ orgName }}</h2>
+          <h2 class="app-nav__header-title u-text--overflow">{{ workspaceName }}</h2>
           <span class="app-nav__header-arrow">{{far "angle-down"}}</span>
         </div>
         <div class="u-text--overflow">{{ userName }}</div>
       {{/if}}
     `,
     templateContext() {
       const currentUser = Radio.request('bootstrap', 'currentUser');
-      const currentOrg = Radio.request('bootstrap', 'organization');
+      const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
 
       return {
         userName: currentUser.get('name'),
-        orgName: currentOrg.get('name'),
+        workspaceName: currentWorkspace.get('name'),
         isMinimized: this.getOption('state').get('isMinimized'),
       };
     },
   },
   picklistOptions() {
     return {
       className: 'picklist app-nav__picklist',
-      itemClassName: 'flex flex-align-center app-nav__picklist-item',
-      lists: [{
-        collection: this.collection,
-        itemTemplate: hbs`
-          {{fa icon.type icon.icon classes=icon.classes~}}
-          <span>{{formatMessage text}}</span>
-        `,
-      }],
     };
   },
+  onPicklistSelect({ model }) {
+    if (model.get('disableOnSelect')) return;
+
+    this.popRegion.empty();
+  },
   picklistEvents: {
     'picklist:item:select': 'onSelect',
   },
   onSelect({ model }) {
-    model.get('onSelect')();
+    if (model.get('onSelect')) model.get('onSelect')();
+  },
+  initialize({ workspacesMenuItems, appNavMenuItems }) {
+    const currentOrg = Radio.request('bootstrap', 'organization');
+    const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
+
+    this.lists = [];
+
+    this.lists.push({
+      collection: new Backbone.Collection([{
+        text: currentOrg.get('name'),
+        disableOnSelect: true,
+      }]),
+      headingText: i18n.mainNavDroplist.organizationHeading,
+      headingClassName: 'app-nav__picklist-heading',
+      itemClassName: 'app-nav__picklist-workspace-name',
```

Comment:
Added a separate classname to the organization name, so I could apply different font size and padding properties.